08.Packages.md

```diff
@@ -0,0 +1,12 @@
+# Packages
+
+Some contributors are creating and maintaining external libraries that could simplify your life on your project. Those contributors created packages that use Gitter API in different shapes (progamming languages, main goals, ...). You can retrieve a list of the current existing packages below, sorted by languages.
+
+## .NET
```

Comment:
In the case you want to say I use C# to create this package, it's ok.
In the case you want to say it can be used in C#, I suppose it's ok too since C# is now the favorite language of .NET developers.